src/content-handlers/iiif/IIIFContentHandler.ts

```diff
@@ -406,6 +406,7 @@ export default class IIIFContentHandler
     } catch (e) {
       this.hideSpinner();
       alert("Unable to load manifest");
+      console.error(e);
```

Comment:
Does this belong here? Seems like a reasonable thing to do, but seems out of scope for this PR.